examples/meta/generator/targets/java.json

```diff
@@ -34,10 +34,22 @@
         "IntMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)",
         "LongIntMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)",
         "ULongIntMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)",
-        "ShortRealMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)",
+        "ShortRealMatrix": "FloatMatrix $name = new FloatMatrix($arguments)",
```

Comment:
I am assuming this should be `FloatMatrix`?

Reply:
not sure what the typemap exactly does...but I think our current Java bindings treat everything as double matrix. Might be worth cleaning that up at some point soon